src/resolve/resolved-link.c

```diff
@@ -1457,3 +1458,55 @@ bool link_negative_trust_anchor_lookup(Link *l, const char *name) {
 
         return false;
 }
+
+int link_dump_configuration_json(Link *l, sd_json_variant **ret) {
```

Comment:
link_dump_configuration_json() and manager_dump_...() has mostly same logic. Maybe, better to extract the logic?

Reply:
Good point. I have consolidated this into a helper function in `resolved-manager.c` instead.